### ISSUE TITLE
LeftCurly not working correctly with lambdas

### ISSUE THREAD

**Issue by mjpt777** [checkstyle/checkstyle#2818]
I'm using 6.14.1 of Checkstyle.

```
    <module name="LeftCurly">
        <property name="option" value="nl"/>
    </module>
```

The above rule does not catch a left curly bracket that is used in a lambda like the following:

```
    (args) -> {
        // some code here.
    }

    // Which should be
    (args) ->
    {
        // some code here
    }    
```

**Comment by romani** [comment]
fix is merged.

**Comment by mjpt777** [comment]
Thanks
